### PR TITLE
[post-tex] Drop PFS support (and some cleanup)

### PR DESCRIPTION
**Pull request by vuntz**
Pull from source hasn't been maintained for quite some time and it's
very likely broken. So drop support for it.

### CODE REVIEW

File: chef/cookbooks/neutron/recipes/network_agents.rb [hunk 1]
```diff
@@ -16,34 +16,13 @@
 
 include_recipe "neutron::common_agent"
 
-unless node[:neutron][:use_gitrepo]
-  pkgs = [ node[:neutron][:platform][:dhcp_agent_pkg], node[:neutron][:platform][:metering_agent_pkg] ]
-  pkgs.uniq.each { |p| package p }
+pkgs = [ node[:neutron][:platform][:dhcp_agent_pkg], node[:neutron][:platform][:metering_agent_pkg] ]
+pkgs.uniq.each { |p| package p }
```
Comment (jsuchome) [inline chef/cookbooks/neutron/recipes/network_agents.rb]: Wouldn't it be cleaner/nicer, and more in line with next package call, just to write
``package node[:neutron][:platform][:dhcp_agent_pkg]``
``package node[:neutron][:platform][:metering_agent_pkg]``

(missing ``uniq`` should not hurt)

**Comment by jsuchome** [comment]
+1 (tested few neutron commands after applying the patch and redeploying barclamp)

File: chef/cookbooks/neutron/recipes/network_agents.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/neutron/recipes/network_agents.rb]: Fixed with a new commit.

**Comment by ovisan** [comment]
:+1:

**Pull request merged**